src/api/index.spec.tsx

```diff
@@ -550,4 +568,120 @@ describe(__filename, () => {
       expect(query).toEqual(expect.urlWithTheseParams({ flag: 'true' }));
     });
   });
+
+  describe('createOrUpdateComment', () => {
+    const _createOrUpdateComment = (params = {}) => {
+      return createOrUpdateComment({
+        _callApi: jest.fn(),
+        addonId: 123,
+        apiState: defaultApiState,
+        cannedResponseId: undefined,
+        comment: 'Example comment',
+        commentId: undefined,
+        fileName: null,
+        line: null,
+        versionId: 321,
+        ...params,
+      });
+    };
+
+    it('can POST a comment', async () => {
```

Comment:
Nit: I wonder if a better name for this test would be "can create a comment" - it's more descriptive and less implementation-y.